examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -2377,17 +2354,17 @@ server cluster OccupancySensing = 1030 {
 }
 
 server cluster OnOff = 6 {
-  enum OnOffDelayedAllOffEffectVariant : enum8 {
+  enum OnOffDelayedAllOffEffectVariant : ENUM8 {
     kFadeToOffIn0p8Seconds = 0;
     kNoFade = 1;
     k50PercentDimDownIn0p8SecondsThenFadeToOffIn12Seconds = 2;
   }
 
-  enum OnOffDyingLightEffectVariant : enum8 {
+  enum OnOffDyingLightEffectVariant : ENUM8 {
     k20PercenterDimUpIn0p5SecondsThenFadeToOffIn1Second = 0;
   }
 
-  enum OnOffEffectIdentifier : enum8 {
+  enum OnOffEffectIdentifier : ENUM8 {
```

Comment:
is there a logic of changing case? I see enum8 became uppercase, while fabric/nodeid and such became lowercase.

If this is for some consistency (internal vs external types) please separate that as a standalone PR.

Reply:
Good point I was a bit confused on how the existing content varied on the case. Would it make sense if I just change all of it to uppercase or lowercase?